hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMFileCreateRequest.java

```diff
@@ -206,6 +208,8 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
           IAccessAuthorizer.ACLType.CREATE, OzoneObj.ResourceType.KEY);
 
       // acquire lock
+      acquireVolumeLock = omMetadataManager.getLock().acquireWriteLock(
+          VOLUME_LOCK, volumeName);
       acquiredLock = omMetadataManager.getLock().acquireWriteLock(BUCKET_LOCK,
```

Comment:
Nit: Can we rename all variable name of acquiredLock to acquireBucketLock?

Reply:
Thanks @linyiqun.  Changed this with new commit.